.mailmap

```diff
@@ -1,9 +1,9 @@
-Stephen J Day <stephen.day@docker.com>  Stephen Day <stevvooe@users.noreply.github.com>
-Stephen J Day <stephen.day@docker.com>  Stephen Day <stevvooe@gmail.com>
-Olivier Gambier <olivier@docker.com>    Olivier Gambier <dmp42@users.noreply.github.com>
-Brian Bland <brian.bland@docker.com>    Brian Bland <r4nd0m1n4t0r@gmail.com>
+Stephen J Day <stephen.day@docker.com> Stephen Day <stevvooe@users.noreply.github.com>
```

Comment:
Non-blocking: This might be a result of tooling used to generate this file but is it ok to provide github.com email address since it is not read?

Reply:
Not quite sure I follow. This file is manually created, as opposed to a file like `AUTHORS` which tends to be generated. This file is formatted to have a single name/email for each individual which has multiple name/email author lines in the git commit. In a perfect world everyone has their `user.email` and `user.name` git configs set to an email which never changes and their real name, and maybe we have entries for maintainers who merge but have github configured to use their no reply email. It is in the contributing docs that real names must be used, but that is hard to enforce on every PR since there is no real way to automate that, the mailmap attempts to clean that up. Unfortunately after we merge there is also no way we can force someone to provide their real name if they didn't before we merged their commit.

Reply:
Cool. Completely agreed with your points. Since this file is created manually do you it is too much to ask to remove no-reply email address since another email of the same user is also there in the file? I am completely ok leaving it like this.